demo/stories/10-announcement.stories.mdx

```diff
@@ -0,0 +1,234 @@
+import { Story, Meta, html, Preview } from '@open-wc/demoing-storybook';
+
+<Meta title="Intro/Announcement" />
+
+# ING Bank open sources Lion
+## A library for building performant, accessible & flexible Web Components
+
+**TL;DR:** Web development is hard, wether you're making your own components, implementing Design Systems, incorporating accessibility, or adding third party dependencies. Lion aims to make your life a little bit easier for you, by taking the ground work of functional, accessible, and performant components out of your hands!
+
+As many of you may already know, ING has a long and rich history of Web Components, starting out with the Polymer library, all the way up to the recently released LitElement.
```

Comment:
`As many of you may already know` -> double again. Would be `As many of you may know` or `As you may already know`
I also wouldn't count on the average reader having read this article. Then it would become `As some of you may know`
Maybe make `history of Web Components` -> `history of building Web Components`